dev/linux/setup.sh

```diff
@@ -6,10 +6,15 @@ apt-get update --fix-missing
 apt-get install -y --no-install-recommends \
     tini wget build-essential bzip2 ca-certificates \
     libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion \
-    sudo htop less nano man grep
+    sudo htop less nano man grep curl
```

Comment:
```suggestion
    sudo htop less nano man grep
```

No need since we already have wget.